client/src/stores/settings.ts

```diff
@@ -8,6 +9,7 @@ export interface SettingsState {
 	theme: Theme;
 	sfxEnabled: boolean;
 	sfxVolume: number;
+	autoSkipSegmentCategories?: Category[];
```

Comment:
For the client settings here, it would be better if we could nest this under a `defaultRoomSettings` so we can add other settings like `grants` to it.